_pages/Dashboard/components/Cards/index.js

```diff
@@ -1,5 +1,7 @@
 import styles from './styles/cards.module.css';
 
+const lang = { mw: 'Modern Warfare', cw: 'Cold War', psn: 'Playstation', wz: 'War Zone', mp: 'Multiplayer' };
```

Comment:
Nice use of `lang`, I'm sure I've seen a similar pattern somewhere before... 😉 

Would be nice to move this into it's own file to reduce noise in this file!